### ISSUE TITLE
Property once not working

### ISSUE THREAD

**Issue by pistou** [jasonslyvia/react-lazyload#10]
I'm currently using react-lazyload v1.3.2.
I'm trying to add the property `once` but it's breaking my code:

```
<Lazyload once>
    <img src="http://lorempixel.com/500/500" className="responsive-img" />
</Lazyload>
```

Gives me an error (at least on Chrome - didn't check on other browsers):

> Uncaught TypeError: Cannot read property 'ownerDocument' of null

This error comes from the function `checkVisible`, at the following lines:
```
var node = _reactDom2['default'].findDOMNode(component);
var parent = _utilsScrollParent2['default'](node);
var isOverflow = parent !== (node.ownerDocument || document); // here
```

Seems like `_reactDom2['default'].findDOMNode(component)` is undefined or null for some reasons.

**Comment by danderu** [comment]
I'm having the same issue :\

**Comment by jasonslyvia** [comment]
> Uncaught TypeError: Cannot read property 'ownerDocument' of null

This error should be fixed in `1.4.0`, however working with image still to be implemented.

**Comment by jasonslyvia** [comment]
I'm closing this and all images related issue can be discussed in #9
